e2e/playwright/point-click.spec.ts

```diff
@@ -3006,151 +3006,7 @@ chamfer(extrude001, length = 5, tags = [getOppositeEdge(seg02)])
-          })
-          await clickOnCap()
-          await page.waitForTimeout(500)
-          await cmdBar.progressCmdBar()
-          await page.waitForTimeout(500)
-          await cmdBar.progressCmdBar()
-          await cmdBar.expectState({
-            stage: 'review',
-            headerArguments: {
-              Selection: '1 cap',
-              Thickness: '5',
-            },
-            commandName: 'Shell',
-          })
-          await cmdBar.progressCmdBar()
-        })
-      } else {
-        await test.step(`Preselect the cap`, async () => {
-          await clickOnCap()
-          await page.waitForTimeout(500)
-        })
-
-        await test.step(`Go through the command bar flow with a preselected face (cap)`, async () => {
-          await toolbar.shellButton.click()
-          await cmdBar.progressCmdBar()
-          await page.waitForTimeout(500)
-          await cmdBar.progressCmdBar()
-          await cmdBar.expectState({
-            stage: 'review',
-            headerArguments: {
-              Selection: '1 cap',
-              Thickness: '5',
-            },
-            commandName: 'Shell',
-          })
-          await cmdBar.progressCmdBar()
-        })
-      }
-
-      await test.step(`Confirm code is added to the editor, scene has changed`, async () => {
-        await editor.expectEditor.toContain(shellDeclaration)
-        await editor.expectState({
-          diagnostics: [],
-          activeLines: [shellDeclaration],
-          highlightedCode: '',
-        })
-        await scene.expectPixelColor([146, 146, 146], testPoint, 15)
-      })
-
-      await test.step('Edit shell via feature tree selection works', async () => {
-        await toolbar.openPane('feature-tree')
-        const operationButton = await toolbar.getFeatureTreeOperation(
-          'Shell',
-          0
-        )
-        await operationButton.dblclick()
-        await cmdBar.expectState({
-          stage: 'arguments',
-          currentArgKey: 'thickness',
-          currentArgValue: '5',
-          headerArguments: {
-            Thickness: '5',
-          },
-          highlightedHeaderArg: 'thickness',
-          commandName: 'Shell',
-        })
-        await page.keyboard.insertText('2')
-        await cmdBar.progressCmdBar()
-        await cmdBar.expectState({
-          stage: 'review',
-          headerArguments: {
-            Thickness: '2',
-          },
-          commandName: 'Shell',
-        })
-        await cmdBar.progressCmdBar()
-        await toolbar.closePane('feature-tree')
-        await scene.expectPixelColor([150, 150, 150], testPoint, 15)
-        await editor.expectEditor.toContain(editedShellDeclaration)
-        await editor.expectState({
-          diagnostics: [],
-          activeLines: [editedShellDeclaration],
-          highlightedCode: '',
-        })
-      })
-    })
-  })
-
-  test('Shell point-and-click wall', async ({
+  test(`Shell point-and-click`, async ({
```

Comment:
Keeping only one e2e test here, the rest of the coverage has moved to `faces.test.ts`